recipes/sardana-alba/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sardana-alba" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ALBA-Synchrotron/sardana-alba/archive/{{ version }}.tar.gz
+  sha256: ff4a9e1d82dfd639168974387f1de4d5bab2636e01764650afdfabe78b6186d8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - sardana
+
+test:
+  imports:
+    - sardana_alba
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ALBA-Synchrotron/sardana-alba
+  summary: Sardana common ALBA plugins
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rhomspuron
+    - ovallcorba
```

Comment:
FYI i think you can do something like `conda-forge/sardana` or something like that if you want
https://github.com/conda-forge/qt-feedstock/blob/main/recipe/meta.yaml#L91

Reply:
Thanks @hmaarrfk this is a good idea.